attributes/network/14_proof.adoc

```diff
@@ -0,0 +1,28 @@
+= Proof Attribute (#14)
+:cddl: ./cddl/
+
+This attribute allows server to deliver the proof of an object requested.
+The proof itself is a sequence of actions used to construct the state in which the object is embedded within the corresponding Merkle tree.
+
+== Attribute Argument
+
+This is a uint representing the hasher to be used in constructing proofs.
+
+* 0 - merk default hash (currently blake3)
```

Comment:
Should point to the `Hash Scheme` section of this document. I have a feeling we'll have more than one later too, so if you want to create a new document and point to it that'd be even better.